libs/dal/src/repositories/job/job.entity.ts

```diff
@@ -16,6 +16,7 @@ export class JobEntity {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   overrides: Record<string, Record<string, unknown>>;
   step: NotificationStepEntity;
+  tenant?: ITenantDefine;
```

Comment:
Added so we can show it in the snippet in activity feed -  so we can show the user a correct and accurate presentation of the snippet he used  + from here we take the identifier for the messages compilation